templates/patient_finder/finder.html.twig

```diff
@@ -38,17 +38,21 @@
                                 <thead class="thead-light">
                                     <tr>
                                         {% for h in headers %}
-                                            <th scope="col">{{ h.title|text }}</th>
+                                            <th scope="col">{{ h.title|xlt }}</th>
```

Comment:
Rather than `xlt`, use `xlListLabel` above. This is because some users will put their translations directly in list_options title and in that case they will turn off translations of lists (a setting in globals).

Otherwise, code looks great!

Reply:
Thank you.  I have pushed this change.  RKL